src/model.jl

```diff
@@ -410,23 +410,23 @@ function get_consumer_benchmark(c::ConsumerRef)
     end
 end
 
```

Comment:
Never gets called, but I wasn't 100% sure this isn't going to be useful at some point so I commented out instead of deleting.